.github/workflows/lint-check.yml

```diff
@@ -17,14 +17,15 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
-        with:
```

Comment:
do we just not have submodules anymore or does the linting not need to happen on them... why no more submodules?